guides/source/association_basics.md

```diff
@@ -110,13 +110,37 @@ class CreateBooks < ActiveRecord::Migration[5.0]
 
     create_table :books do |t|
       t.belongs_to :author, index: true
-      t.datetime :published_at
+      t.string :name
       t.timestamps
     end
   end
 end
 ```
 
+As an example, using the rails console, let's first create an author:
```

Comment:
Can you try and standardize the top language; here it's "using the rails console", below it's "working from the console", etc...